src/nvim/ex_cmds.lua

```diff
@@ -2488,7 +2488,7 @@ return {
   },
   {
     command='source',
-    flags=bit.bor(BANG, FILE1, TRLBAR, SBOXOK, CMDWIN),
+    flags=bit.bor(RANGE, DFLALL, WHOLEFOLD, BANG, FILE1, TRLBAR, SBOXOK, CMDWIN),
```

Comment:
@in42 Since I've never used these, can you explain what they mean to me?

Reply:
These flags are used to specify the behavior of that ex command. E.g. RANGE allows the the ex command to work on a range instead of the whole buffer.
Detailed descriptions of the flags are present in `src/nvim/ex_cmds_defs.h`.

The descriptions for the newly added flags are:
```
#define RANGE           0x001   /* allow a linespecs */
...
#define DFLALL          0x020   /* default file range is 1,$ */
#define WHOLEFOLD       0x040   /* extend range to include whole fold also
                                   when less than two numbers given */
```

Reply:
Gotcha, thank you very much @in42 :)